OpenRA.Mods.Common/Traits/CaptureManager.cs

```diff
@@ -189,7 +189,7 @@ public bool StartCapture(CaptureManager targetManager, out Captures captures)
 			if (target != currentTarget)
 			{
 				if (currentTargetManager != null)
```

Comment:
Prior to #20261 this was checking `currentTarget != null`. Would there be any benefit in changing this back to match?

Reply:
no, manager can't be set if actor wasn't set prior